libcontainer/configs/cgroup_linux.go

```diff
@@ -69,6 +69,9 @@ type Resources struct {
 	// CPU hardcap limit (in usecs). Allowed cpu time in a given period.
 	CpuQuota int64 `json:"cpu_quota"`
 
+	// CPU hardcap burst limit (in usecs). Allowed accumulated cpu time additionally for burst in a given period.
+	CpuBurst uint64 `json:"cpu_burst"` //nolint:revive
```

Comment:
Please change to `CpuBurst *unit64`

Reply:
I think the filed type should be as same as other fields like `CpuQuota`

Reply:
No, For `CpuQuota`, `-` means `no limit`, but for `CpuBurst`, `0` means `no config`.

Reply:
I think `0` is a legal value which should be allowed to write to cgroup.

Reply:
SGTM